test/e2e/suites/e2e/e2e_test.go

```diff
@@ -123,11 +123,10 @@ var _ = Describe("e2e tests [PR-Blocking]", func() {
 			openStackCluster, err := shared.ClusterForSpec(ctx, e2eCtx, namespace)
 			Expect(err).NotTo(HaveOccurred())
 
-			// Tag: clusterName is declared on OpenStackCluster and gets propagated to all machines
-			// except the bastion host
+			// Tag: clusterName is declared on OpenStackCluster and gets propagated to all machines, including the bastion.
 			allServers, err := shared.DumpOpenStackServers(e2eCtx, servers.ListOpts{Tags: clusterName})
 			Expect(err).NotTo(HaveOccurred())
-			Expect(allServers).To(HaveLen(2))
+			Expect(allServers).To(HaveLen(3))
```

Comment:
Should we document this behavior change in the release notes or v1beta1 docs? It's additive, but someone might be depending on this. Obviously the change itself makes perfect sense to me.

Reply:
I created https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/1958